datapipelines/build.gradle

```
@@ -37,4 +37,8 @@ dependencies {
     api project(":core")
     implementation(Dependencies.segment)
     implementation(Dependencies.kotlinxSerializationJson)
+
+    api Dependencies.mockito
+    api Dependencies.mockitoKotlin
+    api Dependencies.robolectric
```

Shouldn't these dependencies be `testImplementation`?

```suggestion
    testImplementation(Dependencies.mockito)
    testImplementation(Dependencies.mockitoKotlin)
    testImplementation(Dependencies.robolectric)
```

---

🤦

---

🤦